<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALCDataLoadingView</class>
 <widget class="QWidget" name="ALCDataLoadingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="dataGroup">
         <property name="title">
          <string>Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="firstLastLayout">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>First</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Last</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="MantidQt::API::MWRunFiles" name="lastRun" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="label" stdset="0">
               <string/>
              </property>
              <property name="multipleFiles" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="lastRunAuto">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked, loads all files in directory&lt;/p&gt;&lt;p&gt;and automatically updates as files are added&lt;/p&gt;&lt;p&gt;and removed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Auto</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="MantidQt::API::MWRunFiles" name="firstRun" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="label" stdset="0">
               <string/>
              </property>
              <property name="multipleFiles" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="MantidQt::MantidWidgets::LogValueSelector" name="logValueSelector" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="deadTimeGroup">
         <property name="title">
          <string>Dead Time Correction</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutDeadTime">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutDeadTime">
            <item>
             <widget class="QRadioButton" name="none">
              <property name="text">
               <string>None</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">deadTimeCorrType</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="fromDataFile">
              <property name="text">
               <string>From Data File</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">deadTimeCorrType</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="fromCustomFile">
              <property name="text">
               <string>From Custom File</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">deadTimeCorrType</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacerDeadTime">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutDeadTime_2">
            <item>
             <widget class="MantidQt::API::MWRunFiles" name="deadTimeFile" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="label" stdset="0">
               <string/>
              </property>
              <property name="multipleFiles" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="detectorGroupingGroup">
         <property name="title">
          <string>Grouping</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutGrouping">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutGrouping">
            <item>
             <widget class="QRadioButton" name="autoGroupingButton">
              <property name="text">
               <string>Auto</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">detectorGroupingType</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="customGroupingButton">
              <property name="text">
               <string>Custom</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">detectorGroupingType</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacerGrouping">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutGrouping_2">
            <item>
             <widget class="QLabel" name="labelForward">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Forward</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="forwardEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelBackward">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Backward</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="backwardEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="periodsGroup">
         <property name="title">
          <string>Periods</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutPeriods">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutPeriods">
            <item>
             <widget class="QComboBox" name="redPeriod"/>
            </item>
            <item>
             <widget class="QCheckBox" name="subtractCheckbox">
              <property name="text">
               <string>Subtract</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="greenPeriod">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hSpacerPeriods">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="calculationGroup">
         <property name="title">
          <string>Calculation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="integral">
              <property name="text">
               <string>Integral</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">calculationType</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="differential">
              <property name="text">
               <string>Differential</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">calculationType</string>
              </attribute>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="timeLimits" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>From [µ]</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="minTime">
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="minimum">
                  <double>-10.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>50.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Max [µ]</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="maxTime">
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="minimum">
                  <double>-10.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>50.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="help">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="load">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Loaded data</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QwtPlot" name="dataPlot">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::API::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/MWRunFiles.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::LogValueSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/LogValueSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>integral</tabstop>
  <tabstop>differential</tabstop>
  <tabstop>minTime</tabstop>
  <tabstop>maxTime</tabstop>
  <tabstop>help</tabstop>
  <tabstop>load</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fromCustomFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>deadTimeFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customGroupingButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelForward</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customGroupingButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelBackward</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customGroupingButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>forwardEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customGroupingButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>backwardEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtractCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>greenPeriod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="detectorGroupingType"/>
  <buttongroup name="calculationType"/>
  <buttongroup name="deadTimeCorrType"/>
 </buttongroups>
</ui>
